### PR TITLE
Deploy documentation to github pages

### PR DESCRIPTION
**Pull request by andrewspiers**
The idea here is to have Travis build the documentation and publish it to a separate repo, to be
created under the CSIRO-Precision-Agriculture organisation.

### CODE REVIEW

**Comment by AdrianDAlessandro** [comment]
I've pushed up a change that allows the kriging_ops docs to render

**Comment by AdrianDAlessandro** [comment]
> The docs build correctly and are pushed once Travis passes. Just two things I think should be addressed before merging:
> 
> 1. Make the separate docs repository under this organisation
> 2. Make the docs still build even if the opengeo servers can't be accessed and we get the timeout error

I missed that Christina has just made that change for the opengeo seervers-reliant test in PR "Fixed CRS Tests"

Now only need to make the separate repo

**Comment by andrewspiers** [comment]
I just wanted to check that everyone was ok with me creating a separate repo for the docs before I went ahead, so will sort that out today.

**Comment by andrewspiers** [comment]
There is a built copy of the docs currently at https://andrewspiers.github.io/pyprecag_doc/

**Comment by dgobbett** [comment]
Looks good!  LICENSE file doesn't get included in the License page

**Comment by AdrianDAlessandro** [comment]
Well spotted @dgobbett ! That's revealed a bigger issue I hadn't noticed.
None of the top level files are being included

**Comment by andrewspiers** [comment]
from the travis logs:
```
/docs/authors.rst:1: WARNING: Problems with "include" directive path:
InputError: [Errno 2] No such file or directory: '/AUTHORS.rst'.
/docs/changelog.rst:1: WARNING: Problems with "include" directive path:
InputError: [Errno 2] No such file or directory: '/CHANGELOG.rst'.
/docs/contributing.rst:1: WARNING: Problems with "include" directive path:
InputError: [Errno 2] No such file or directory: '/CONTRIBUTING.rst'.
/docs/installation.rst:22: WARNING: Literal block expected; none found.
/docs/license.rst:4: WARNING: Problems with "include" directive path:
InputError: [Errno 2] No such file or directory: '/LICENSE'.
/docs/installation.rst:22: WARNING: Literal block expected; none found.
/docs/contributing.rst:1: WARNING: Problems with "include" directive path:
InputError: [Errno 2] No such file or directory: '/CONTRIBUTING.rst'.
/docs/reference/pyprecag.bandops.rst:2: WARNING: Title underline too short.

```

File: README.rst [hunk 1]
```diff
@@ -6,7 +6,7 @@ A suite of tools for Precision Agriculture
 
 * Free software: CSIRO Open Source Software License Agreement
 * Homepage: https://github.com/CSIRO-Precision-Agriculture/pyprecag
-* Documentation: https://pyprecag.readthedocs.org
+* Documentation: https://CSIRO-Precision-Agriculture.github.io/pyprecag_docs
```
[the remark this answers is not in the document]
Comment (AdrianDAlessandro) [reply README.rst]: Yes, it is correct. This is where GitHub Pages will render the documentation. The link you provided will be the repository

**Pull request merged**